server/src/index.ts

```diff
@@ -13,7 +13,13 @@ import logger from "./logging";
 const port = 8000;
 const app = express();
 const httpServer = createServer(app);
-const io = new Server(httpServer);
+const io = new Server(httpServer, {
+  cors: {
+    origin: "http://localhost:8001",
+    methods: ["GET", "POST"],
+    credentials: true,
```

Comment:
cors needed to be enabled to connect to the UI. This should be configured at build time, similar to what i'm doing in `socketConnection.ts` below